android-client/Espoo/.idea/deploymentTargetDropDown.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="deploymentTargetDropDown">
+    <runningDeviceTargetSelectedWithDropDown>
+      <Target>
+        <type value="RUNNING_DEVICE_TARGET" />
+        <deviceKey>
+          <Key>
+            <type value="VIRTUAL_DEVICE_PATH" />
+            <value value="C:\Users\ami_a\.android\avd\Pixel_2_API_30.avd" />
```

Comment:
Looks like this is some personal/local config. It should not be here.

Reply:
nani

Reply:
this can to be a variable of enviroment? @amiapmorais

Reply:
As this file is inside .idea folder, I assume it is IDE config, maybe .ignore for it

Reply:
I guess the solution is put .idea folder in gitignore file

Reply:
Agree @amiaram

Reply:
@amiapmorais , the only missing change to merge this request is to ignore .idea folder, correct? could you do it so we merge it?

Reply:
You can delete this PR, because it was make in the upstream repository. I'm working in my repository fork and soon I will open a new request with this fix already done